File: 3005. Count Elements With Maximum Frequency.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        count = Counter(nums)
        maxCount = max([count[val] for val in count])
        return sum([maxCount for val in count if maxCount == count[val]])



class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        maxCount = 0
        count = Counter(nums)
        for val in count:
            maxCount = max(maxCount, count[val])
        
        ans = 0
        for val in count:
            if count[val] == maxCount:
                ans += maxCount
        return ans

File: Counting/3005. Count Elements With Maximum Frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int> &nums)
    {
        vector<int> freq(105);
        int n = nums.size(), mx = 0;
        for(int i = 0; i < n; i++)
        {
            freq[nums[i]]++;
            mx = max(mx, freq[nums[i]]);
        }

        int ans = 0;
        for(auto x : freq)
            if(x == mx) ans += x;
        return ans;
    }
};